PROJECT

```diff
@@ -37,4 +37,114 @@ resources:
     defaulting: true
     validation: true
     webhookVersion: v1
+- api:
+    crdVersion: v1
+  controller: true
+  domain: opendatahub.io
+  group: components
+  kind: Dashboard
+  path: github.com/opendatahub-io/opendatahub-operator/v2/apis/components/v1
```

Comment:
not in this file, but one thing we can add back is the `operators.operatorframework.io/internal-objects: '[dashboard.opendatahub.io]'`  etc with all these internal CRs